apps/src/lib/levelbuilder/unit-editor/CloneLessonDialog.jsx

```diff
@@ -4,6 +4,11 @@ import BaseDialog from '@cdo/apps/templates/BaseDialog';
 import Button from '@cdo/apps/templates/Button';
 import DialogFooter from '@cdo/apps/templates/teacherDashboard/DialogFooter';
 
+const LevelCopyText = {
+  deepCopy: 'deep copy: create a new copy of each level in the lesson',
```

Comment:
These dropdown options are very long. Could we use a HelpTip to explain what deep and shallow copy mean in order to keep the dropdown options shorter?

Reply:
a dropdown in a dialog wasn't working for me. how about this? cc @tess323 

![Screen Shot 2022-03-21 at 12 57 47 PM](https://user-images.githubusercontent.com/8001765/159354025-a23e133f-b39f-43fa-a4e4-e082db79b35c.png)

Reply:
Would it work to put the table behind a '?' tooltip?  If not I am not I am totally fine with this.

Reply:
a tooltip isn't working for me, due to it being in a dialog, unfortunately.

Reply:
makes total sense - this works for me!